### ISSUE TITLE
Time of day reset to day - Player Interaction

### ISSUE THREAD

**Issue by mickplouffe** [EssentialsX/Essentials#2536]
### Information

**Full output of `/ess version`**:

```

[18:22:29] [Server thread/INFO]: CONSOLE issued server command: /ess version
--
[18:22:29] [Server thread/INFO]: Server version: 1.14-R0.1-SNAPSHOT git-Spigot-4d24e36-ba2e5e7 (MC: 1.14)
[18:22:29] [Server thread/INFO]: EssentialsX version: 2.16.1.153
[18:22:29] [Server thread/INFO]: LuckPerms version: 4.4.1
[18:22:29] [Server thread/INFO]: Vault version: 1.7.2-b107
[18:22:29] [Server thread/INFO]: EssentialsXChat version: 2.16.1.153
[18:22:29] [Server thread/INFO]: You are running an unsupported server version!


```

**Server log**: https://gist.github.com/mickplouffe/501fed4324511dcb17280a8eb9a744db

**EssentialsX config**: https://gist.github.com/mickplouffe/501fed4324511dcb17280a8eb9a744db

### Details

**Description**  
The time of the day resets to morning every time some action/command is done:
Entering AFK
Existing AFK
Someone Die
Someone Join the server
etc

**Steps to reproduce**  
Use /afk,
Die / Respawn
Join the Server
Go to Nether
etc

**Expected behavior**  
Not setting the day when not supposed.

**Comment by mdcfe** [comment]
You didn't provide the output of `/ess version` and so there isn't enough information for us to help. Run `/ess version` and give us the **exact output**.

This appears to be a duplicate of #2516.

**Comment by mickplouffe** [comment]
> You didn't provide the output of `/ess version` and so there isn't enough information for us to help. Run `/ess version` and give us the **exact output**.
> 
> This appears to be a duplicate of #2516.

Yes it seems to be the same Issue. But there is not only the gamemode. (Wich work on my side.)

**Comment by ghost** [comment]
I experience this issue as well on our server, and it appears to start doing this at random sometimes. Suddenly any command or action can cause the time to reset, skipping to the next day.

**Comment by ghost** [comment]
An update from what I've seen: This issue seems to be non-existent in 1.14.1 Spigot. Haven't experienced it once and before it would usually happen often. Seems something was fixed in Spigot/Bukkit when going to 1.14.1. Of course, I am only one person with one server.
EDIT: And of course the linked issue #2516 has been closed for the same reason.

**Comment by triagonal** [comment]
Closing as issue appears to be fixed in Spigot 1.14.1 and above.
